File: ympa_fsharp_functions_server/src/model/PickupAddressDTO.fs
namespace ympa_fsharp_functions_server.Model

open System
open System.Collections.Generic
open Newtonsoft.Json

module PickupAddressDTO =

  //#region PickupAddressDTO

  [<CLIMutable>]
  type PickupAddressDTO = {
    [<JsonProperty(PropertyName = "country")>]
    Country : string;
    [<JsonProperty(PropertyName = "city")>]
    City : string;
    [<JsonProperty(PropertyName = "street")>]
    Street : string;
    [<JsonProperty(PropertyName = "house")>]
    House : string;
    [<JsonProperty(PropertyName = "postcode")>]
    Postcode : string;
  }

  //#endregion

File: src/main/scala/org/openapitools/server/model/PickupAddressDTO.scala
package org.openapitools.server.model


/**
 * Адрес доставки.
 *
 * @param country Страна. for example: ''Россия''
 * @param city Город. for example: ''Москва''
 * @param street Улица. for example: ''Стрелецкая улица''
 * @param house Номер дома. for example: ''9к2''
 * @param postcode Почтовый индекс. for example: ''123518''
*/
final case class PickupAddressDTO (
  country: Option[String] = None,
  city: Option[String] = None,
  street: Option[String] = None,
  house: Option[String] = None,
  postcode: Option[String] = None
)

